### PR TITLE
Include missing in exit code

### LINKED ISSUE #15: Non-zero exit code on missing
Right now, it seems like only `failing` tests get a non-zero exit code. However, when running in pre-commit hooks or CI, I would like missing tests to also give a non-zero exit code since it likely means I either forgot to add an expect file, or forgot to check if the missing tests are fine

### PR DESCRIPTION
**Pull request by TheZoq2**
Closes #15 

I tested this manually on `cli-test`, but as far as I can tell, the output of cli-test is not checked automatically? Also, its version is `0.3.4` not `0.4.0`

### CODE REVIEW

File: src/executor/context.rs [hunk 1]
```diff
@@ -211,6 +211,11 @@ impl Context {
         st.print_summary().await?;
         println!();
 
-        Ok((st.timeout + st.fail) as i32)
+        match opts.post_filter {
+            Some(cli::OnlyOpt::Fail) => Ok((st.fail + st.timeout) as i32),
+            Some(cli::OnlyOpt::Missing) => Ok((st.miss) as i32),
+            Some(cli::OnlyOpt::Pass) => Ok(0),
+            None => Ok((st.fail + st.timeout + st.miss) as i32)
+        }
```
Comment (rachitnigam) [inline src/executor/context.rs:219]: Awesome!!

**Review by rachitnigam — COMMENTED** [review]
Looks great!

**Comment by rachitnigam** [comment]
Version `0.4.1` released with these changes!
